hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -274,11 +314,150 @@ yield shouldThrottleEthTxn(
         };
     }
 
+    private boolean shouldThrottleScheduleCreate(
+            final ThrottleReqsManager manager,
+            final TransactionInfo txnInfo,
+            final Instant now,
+            final HederaState state) {
+        final var txnBody = txnInfo.txBody();
+        final var scheduleCreate = txnBody.scheduleCreate();
+        final var scheduled = scheduleCreate.scheduledTransactionBody();
+        final var schedule = Schedule.newBuilder()
+                .originalCreateTransaction(txnBody)
+                .payerAccountId(txnInfo.payerID())
+                .scheduledTransaction(scheduled)
+                .build();
+
+        TransactionBody innerTxn;
+        HederaFunctionality scheduledFunction;
+        try {
+            innerTxn = childAsOrdinary(schedule);
+            scheduledFunction = functionOf(innerTxn);
+        } catch (HandleException | UnknownHederaFunctionality ex) {
+            log.debug("ScheduleCreate was associated with an invalid txn.", ex);
+            return true;
+        }
+
+        // maintain legacy behaviour
+        final var configuration = configProvider.getConfiguration();
+        final boolean areLongTermSchedulesEnabled =
+                configuration.getConfigData(SchedulingConfig.class).longTermEnabled();
+        log.warn("Long term scheduling is {}.", areLongTermSchedulesEnabled ? "enabled" : "disabled");
```

Comment:
We don't want a warning under normal conditions.  This can be in the else of the conditional on the next line and just warn if long term is enabled to inform "throttling of long term schedules is not yet implemented".

Reply:
Updated, moved warnings to else branch i.e. when long term is enabled.